public/app/themes/justice/inc/admin.php

```diff
@@ -36,4 +37,14 @@ public static function removeCustomizer()
         $customizer_url = add_query_arg('return', urlencode(remove_query_arg(wp_removable_query_args(), wp_unslash($_SERVER['REQUEST_URI']))), 'customize.php');
         remove_submenu_page('themes.php', $customizer_url);
     }
+    
+    /**
+     * Increase the dropdown limit on the block editor parent page field
+     */
+    function increaseDropdownLimit( $args ) {
+        if ( is_user_logged_in() ) {
```

Comment:
Would it be possible to add another check here to make the page increase more targeted?

For example you could access the $request parameter, and use `$request->get_query_params()['context'], it will equal `edit` in this case.

This is a good resource for getting info out of the request.

https://developer.wordpress.org/reference/classes/wp_rest_request/

I think it could help prevent a confusing bug further down the line where we're wondering why our other ajax requests are maxing out at 2000 😆